admin/modules/user/users.php

```diff
@@ -4033,6 +4037,8 @@ function output_custom_profile_fields($fields, $values, &$form_container, &$form
 				}
 				break;
 			case "checkbox":
+				
+				$select_options = array();
```

Comment:
Not sure why this has been moved here though. It was fine where it was before

Reply:
For consistency. Isn't it easier for you to just reformat this however you want? 4 formatting commits for a 4 line fix is just odd. You can edit my changes right?

Reply:
Ah alright, no actions required then :) Unfortunately I can't edit your changes as they are in your fork, and I don't have access to it. Don't worry about having many commits, as we stash them when merging, so they will be all compressed into a single one.

Reply:
Bump :)

Reply:
@Shade- You should be able to add commits to this branch if @fluxtion has [allowed edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/#enabling-repository-maintainer-permissions-on-existing-pull-requests).

Reply:
@JordanMussi I have indeed but since @Shade- isn't around, I'm just going to make the changes so we can finally merge this. I don't like having so many open PRs that are completely ready for merging.